pkg/virt-controller/services/template_test.go

```diff
@@ -156,6 +156,24 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Affinity).To(BeEquivalentTo(&kubev1.Affinity{PodAntiAffinity: &podAntiAffinity}))
 			})
 
+			It("should add tolerations to pod", func() {
```

Comment:
We will also need a functional test. We have some tests already in `tests/vmi_lifecycle_test.go` which use affinity. We can add one for tolerations. What we can do is the following:

 * Temporarily add a taint on a node
 * Run the vm once with the toleration (with affinity for the same node with the taint and check that it can be scheduled)
 * Run the vm once without the toleration (with affinity for the same node with the taint and check that it can not be scheduled)